internal/scorecard/scorecard.go

```diff
@@ -140,10 +141,21 @@ func ScorecardTests(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if err := printPluginOutputs(scViper.GetString(schelpers.VersionOpt), pluginOutputs); err != nil {
+	output, err := printPluginOutputs(scViper.GetString(schelpers.VersionOpt), pluginOutputs)
+	if err != nil {
 		return err
 	}
 
+	apiVersion := scViper.GetString(schelpers.VersionOpt)
+	if schelpers.IsV1alpha2(apiVersion) {
+		testOutput := output.(scapiv1alpha2.ScorecardOutput)
```

Comment:
This might be getting into out-of-scope refactor territory, but it's a little odd to me that we would need the result of a function that prints output to determine the state of our test run.

Is it possible to use `pluginOutputs` here and keep the signature of `printPluginOutputs()` as is?